profiles/serializers.py

```diff
@@ -1,20 +1,81 @@
 """
 Serializers for user profiles
 """
-from rest_framework.serializers import ModelSerializer, SerializerMethodField
+from django.db import transaction
 
-from profiles.models import Profile
+from rest_framework.exceptions import ValidationError
+from rest_framework.serializers import ModelSerializer, SerializerMethodField, IntegerField
+
+from profiles.models import Employment, Profile
+
+
+def save_or_update_work_history(validated_data, profile_id):
+    """
+    create or update employment history for given profile id.
+
+    Args:
+        validated_data: Dict of profile data.
```

Comment:
You need to put the types within parentheses, ie: `validated_data (dict): Dict of profile data`